packages/x-components/src/design-system/utilities/background-color.scss

```diff
@@ -1,23 +1,7 @@
-.x-background {
-  &--lead {
-    background-color: var(--x-color-base-lead) !important;
-  }
-  &--auxiliary {
-    background-color: var(--x-color-base-auxiliary) !important;
-  }
-  &--neutral-10 {
-    background-color: var(--x-color-base-neutral-10) !important;
-  }
-  &--neutral-35 {
-    background-color: var(--x-color-base-neutral-35) !important;
-  }
-  &--neutral-70 {
-    background-color: var(--x-color-base-neutral-70) !important;
-  }
-  &--neutral-95 {
-    background-color: var(--x-color-base-neutral-95) !important;
-  }
-  &--neutral-100 {
-    background-color: var(--x-color-base-neutral-100) !important;
+@import '../base/colors.tokens';
```

Comment:
While I love the idea of automating this (finally), importing token files from other files will generate the variables each time it is imported. Extracting the size and color maps to files which only contain pure SCSS variables should be enough.

![image](https://user-images.githubusercontent.com/68222542/146523217-6c9faf86-a1cf-402f-93dc-2dd5a4451084.png)
![image](https://user-images.githubusercontent.com/68222542/146523320-a0defe80-18f8-458f-938d-da2552416c37.png)
![image](https://user-images.githubusercontent.com/68222542/146523404-6a95cdf6-47b7-43b9-8c3e-65ee5fb5e7e7.png)

Reply:
Even if I don't know how I managed to make `@use` work, it still doesn't help us. If we have our variables or mixins defined in a file which renders real CSS content, `rollup-plugin-styles` will duplicate these styles as many times as they are imported.